CONTRIBUTING.md

```diff
@@ -6,9 +6,7 @@ If you have questions about the library, found a bug or want to suggest a featur
 
 ## Documentation
 
-You can visit the [docs website](https://json-schema-form.vercel.app/), however its source is not in the repo yet, only the build (`/docs_build`). Our docs are still coupled to Remote's internal Design System and integration tests. The effort to decouple it at the moment is too high.
-
-So our strategy is to at each new PR, to update the docs internally and manually copy its build to this repo.
+You can visit the [docs website](https://json-schema-form.vercel.app/), however its source is not in the repo yet. Our docs are still coupled to Remote's internal Design System and integration tests. The effort to decouple it at the moment is too high.
```

Comment:
Perfect this update.

Regarding the commit deleting the docs, could you amend it to be co-authored,
the same way it was done with the [commit adding them](https://github.com/remoteoss/json-schema-form/commit/2ee0a7b9283005fc71e60a9742e7b328e6194879)?

Basically you need to add to the commit body the same lines as in the example above.

Reply:
Approved.

Remember to change the PR title to follow conventional commits.